build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -13,3 +13,14 @@ spec:
     - 172.26.0.0/16
     - 172.36.0.0/16
   defaultSubnetAccessMode: Private
+---
+# Sample to create VPCNetworkConfiguration CR using a pre-created NSX VPC.
+apiVersion: nsx.vmware.com/v1alpha1
+kind: VPCNetworkConfiguration
+metadata:
+  name: vpc-network-config-with-pre-created-vpc
+spec:
+  nsxtProject: proj-2
+  vpc: /orgs/default/projects/proj-1/vpcs/vpc-1
```

Comment:
we do not nsxtProject param since the we have the whole vpc path.

Reply:
Removed accordingly.